pandas/core/shared_docs.py

```diff
@@ -213,7 +213,7 @@
     Name to use for the 'variable' column. If None it uses
     ``frame.columns.name`` or 'variable'.
 value_name : scalar, default 'value'
-    Name to use for the 'value' column.
+    Name to use for the 'value' column. Can't be of an existing column name.
```

Comment:
Minor nitpick - should be `cant be an existing column name`

Reply:
Thanks, working on it.